<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>503</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="verticalLayout_main">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_objects">
        <item>
         <layout class="QGridLayout" name="gridLayout_objects">
          <item row="1" column="0">
           <widget class="QLabel" name="label_Ident">
            <property name="text">
             <string>Ident</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_ident_attribute">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>Attribute</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::LogicalMoveStyle</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_ident_pSet">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>PropertySet</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_object_name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="text">
             <string>Object</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEdit_ident_value">
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>Value</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_object_button">
            <item>
             <widget class="QLineEdit" name="lineEdit_object_name">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Normal</enum>
              </property>
              <property name="placeholderText">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_objects_add">
              <property name="text">
               <string>Add</string>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_object">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <attribute name="headerShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Objects</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Identifier</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Optional</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_property_set">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <attribute name="title">
         <string>PropertySet</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0" colspan="2">
          <widget class="QSplitter" name="splitter_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QTableWidget" name="table_pset">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>Test</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>PropertySet</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>InheritedBy</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Optional</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>BestandsdatenqVerkehrsanlagen</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>Verkehrsanlagen</string>
             </property>
            </item>
           </widget>
           <widget class="QTableWidget" name="table_attribute">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Data Format</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Format</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Value</string>
             </property>
            </column>
           </widget>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_pSet_button">
           <item>
            <widget class="QLabel" name="label_pSet_name">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_pSet_name">
             <property name="frame">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_Pset_add">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_code">
        <attribute name="title">
         <string>Code</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QWidget" name="widget_vertical_stack" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_script_title">
              <property name="text">
               <string>Scripts</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget_scripts">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>161</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
              </property>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton_add_script">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>21</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_delete_script">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_import_script">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>DL</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButton_burger">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>B</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_left">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>&lt;-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_right">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>-&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_script_name">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>2</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="code_edit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1265</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="action_abbreviation_json"/>
     <addaction name="action_allplan"/>
     <addaction name="action_card1"/>
     <addaction name="action_vestra"/>
     <addaction name="action_excel"/>
    </widget>
    <addaction name="action_file_new"/>
    <addaction name="action_file_Open"/>
    <addaction name="action_file_Save"/>
    <addaction name="action_file_Save_As"/>
    <addaction name="action_settings"/>
    <addaction name="menuExport"/>
    <addaction name="action_mapping"/>
   </widget>
   <widget class="QMenu" name="menuDesite">
    <property name="title">
     <string>Desite</string>
    </property>
    <addaction name="action_desite_export"/>
    <addaction name="action_export_bs"/>
    <addaction name="action_export_bookmarks"/>
    <addaction name="action_export_boq"/>
    <addaction name="action_mapping_script"/>
   </widget>
   <widget class="QMenu" name="menuPredefined_Psets">
    <property name="title">
     <string>Predefined Psets</string>
    </property>
    <addaction name="action_show_list"/>
   </widget>
   <widget class="QMenu" name="menuShow_Graphs">
    <property name="title">
     <string>Aggregation</string>
    </property>
    <addaction name="action_show_graphs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDesite"/>
   <addaction name="menuPredefined_Psets"/>
   <addaction name="menuShow_Graphs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_file_new">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_file_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_file_Save_As">
   <property name="text">
    <string>Save As ...</string>
   </property>
  </action>
  <action name="action_file_Open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_desite_Settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_desite_export">
   <property name="text">
    <string>Export Modelcheck</string>
   </property>
  </action>
  <action name="action_show_list">
   <property name="text">
    <string>Show List</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_export_bs">
   <property name="text">
    <string>Export BS</string>
   </property>
  </action>
  <action name="action_export_bookmarks">
   <property name="text">
    <string>Export Bookmarks</string>
   </property>
  </action>
  <action name="action_show_graphs">
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="action_export_boq">
   <property name="text">
    <string>Export  for BoQ</string>
   </property>
  </action>
  <action name="action_mapping_options">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="action_mapping">
   <property name="text">
    <string>Mapping</string>
   </property>
  </action>
  <action name="action_shared_parameter">
   <property name="text">
    <string>Shared Parameters</string>
   </property>
  </action>
  <action name="action_ifc_mapping">
   <property name="text">
    <string>IFC Mapping</string>
   </property>
  </action>
  <action name="action_mapping_script">
   <property name="text">
    <string>Mapping Script</string>
   </property>
  </action>
  <action name="action_abbreviation_json">
   <property name="text">
    <string>Abbreviation JSON</string>
   </property>
  </action>
  <action name="action_allplan">
   <property name="text">
    <string>Allplan</string>
   </property>
  </action>
  <action name="action_card1">
   <property name="text">
    <string>CARD1</string>
   </property>
  </action>
  <action name="action_vestra">
   <property name="text">
    <string>Verstra</string>
   </property>
  </action>
  <action name="action_excel">
   <property name="text">
    <string>Excel</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit_ident_pSet</tabstop>
  <tabstop>lineEdit_ident_attribute</tabstop>
  <tabstop>lineEdit_ident_value</tabstop>
  <tabstop>button_objects_add</tabstop>
  <tabstop>tree_object</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButton_delete_script</tabstop>
  <tabstop>pushButton_import_script</tabstop>
  <tabstop>pushButton_burger</tabstop>
  <tabstop>pushButton_left</tabstop>
  <tabstop>listWidget_scripts</tabstop>
  <tabstop>pushButton_add_script</tabstop>
  <tabstop>pushButton_right</tabstop>
  <tabstop>code_edit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
